utils/model_uploader/model_listing/pretrained_model_listing.json

```diff
@@ -0,0 +1,78 @@
+[
```

Comment:
Let's create a separate PR for model listing related works as we are making the workflow separate now.

Reply:
Sure!